README.md

```diff
@@ -47,9 +47,12 @@ individual project sites when necessary.
 + Python 2.6 or 2.7 (use 32 bit if you're on Windows see `Alternate steps for Windows` section below)
 + pip  (If you use virtualenv (see below) this will be installed automatically)
 + CouchDB >= 1.0 (1.2 recommended) (install from OS package manager (`sudo apt-get install couchdb`) or [here][couchdb])
-  - Mac users: note that when installing erlang, you do NOT need to check out an older version of erlang.rb
+   For Mac users
+       - use $ brew install couchdb
+       - note that when installing erlang, you do NOT need to check out an older version of erlang.rb
+      
 + PostgreSQL >= 9.4 - (install from OS package manager (`sudo apt-get install postgresql`) or [here][postgres])
-+ [Elasticsearch][elasticsearch] 0.90.13. In Ubuntu and other Debian derivatives,
++ [Elasticsearch][elasticsearch] 1.7.4. In Ubuntu and other Debian derivatives,
   [download the deb package][elasticsearch], install, and then **hold** the version to prevent automatic upgrades:
 
         $ sudo dpkg -i elasticsearch-0.90.13.deb
```

Comment:
I don't know what the exact 1.7 equivalent is for the deb package, can anyone check and update it?